content/docs/0.8.x/continuous_delivery/expose_services/index.md

```diff
@@ -59,12 +59,15 @@ kubectl apply -f gateway-manifest.yaml
     INGRESS_PORT=<PORT_OF_YOUR_INGRESS> 
     INGRESS_PROTOCOL=<PROTOCOL>                            # "http" or "https"
     ISTIO_GATEWAY=<GATEWAY_NAME>.<NAMESPACE_OF_GATEWAY>  # e.g. public-gateway.istio-system
+    HOSTNAME_TEMPLATE=<HOSTNAME_TEMPLATE> # optional, default = \${INGRESS_PROTOCOL}://\${service}.\${project}-\${stage}.\${INGRESS_HOSTNAME}:\${INGRESS_PORT}
```

Comment:
Looking at the [implementation (GetPublicDeploymentURI function)](https://github.com/keptn/keptn/pull/2882/files#diff-1b85315ea444d67cd1500303c4742f473cfedd12fb634fbfb2bdd5fc47cae4e1R48-R59), I believe it should be `INGRESS_HOSTNAME_SUFFIX`, not `INGRESS_HOSTNAME` (which is the variable defined above).

Maybe we just rename the variable aswell in here!

Reply:
good catch, I will adapt that